app/views/api/v1/books/index.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.books @books.each do |book|
+  json.title  book.title
```

Comment:
You can create a partial `_book.json.jbuilder` containing:

```json.title  book.title
json.author book.author.to_s```

And here only have:

```json.books @books, partial: 'book', as: :book```